pages/solid-events.md

```diff
@@ -14,6 +14,7 @@ Are you interested in speaking about what you have been building on Solid? Reach
 
 | Date | Event | Organiser | Recordings, Slides and Notes |
 |----|-------|-------------|------------------------------|
+| 2021-02-04 | [Solid World February](https://www.eventbrite.com/e/solid-world-tickets-135541896461?ref=estw)|[Solid Team](https://solidproject.org/team)||
```

Comment:
```suggestion
| 2021-02-04 | [Solid World February](https://www.eventbrite.com/e/solid-world-tickets-135541896461)|[Solid Team](https://solidproject.org/team)||
```
I'm not sure what this query param is used for, probably some kind of tracking, and anyways it's not required for the link to work :)